testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -1826,6 +1826,28 @@ QUnit.test("T277051 - the 'e' letter entered in the center of text should not be
     assert.equal(numberBox.option("value"), 900000, "value is correct");
 });
 
+QUnit.test("Should ignore backspace/delete key down when the caret in the start/end of input (T713045)", (assert) => {
```

Comment:
This test should be placed to the `mask.tests.ks` file. In that file the test can be simplified